imports/plugins/core/orders/client/templates/list/ordersList.js

```diff
@@ -1,12 +1,18 @@
 import moment from "moment";
 import { Template } from "meteor/templating";
 import { Orders, Shops } from "/lib/collections";
+import "./cancelOrder.css";
 
 /**
  * dashboardOrdersList helpers
  *
  */
 Template.dashboardOrdersList.helpers({
+  orderCancelled() {
+    if (this.workflow.status === "cancelled") {
```

Comment:
`return this.workflow.status === "cancelled"` should work fine without the if block